cdap-common/src/main/java/co/cask/cdap/common/lang/ProgramResources.java

```diff
@@ -115,8 +115,8 @@ public String apply(ClassPath.ClassInfo input) {
     Set<String> resources = getBaseResources();
 
     // Base on the type, add extra resources
-    // Current only Spark type has extra visible resources
-    if (type == ProgramType.SPARK) {
+    // Current only Spark and Workflow type has extra visible resources
+    if (type == ProgramType.SPARK || type == ProgramType.WORKFLOW) {
```

Comment:
We have support for Spark in Workflow from a long time and it works fine as far as I know. We also did demos where Spark was running as a part of Workflow in Spark summit. Do you mean it lacks some feature inside workflow ?